### ISSUE TITLE
General question

### ISSUE THREAD

**Issue by sainimohit23** [RaptorMai/Deep-AutoEncoder-Recommendation#1]
Hi Zheda,

I want to know that is this project based on this paper: https://arxiv.org/abs/1708.01715 and it's official implementation in pytorch?

I also want to know that have you implemented re-feeding?

**Comment by RaptorMai** [comment]
> Hi Zheda,
> 
> I want to know that is this project based on this paper: https://arxiv.org/abs/1708.01715 and it's official implementation in pytorch?
> 
> I also want to know that have you implemented re-feeding?

**Comment by sainimohit23** [comment]
@RaptorMai I couldn't find any way to implement re-feeding in keras. So, I have implemented it in tensorflow. I have a silly question about re-feeding that I have asked here: https://github.com/NVIDIA/DeepRecommender/issues/29 . Can you answer that?

**Comment by RaptorMai** [comment]
>In dense re-feeding what do we show as output to model. For example let's say B is our training batch. For model input X and label y both will be equal to B. Let's say model gives y_hat as output for our batch B. Now, in dense re-feeding we give this y_hat as input to model. Now my question is what will be the labels in dense re-feeding? Will it be y_hat or is it B?

It should be B.
